one/src/daemon.rs

```diff
@@ -236,6 +237,33 @@ pub struct DaemonOpts {
         env = "CERAMIC_ONE_ETHEREUM_RPC_URLS"
     )]
     ethereum_rpc_urls: Vec<String>,
+
+    /// Enable the aggregator, requires Flight SQL to be enabled
+    #[arg(
+        long,
+        requires = "flight_sql_bind_address",
+        env = "CERAMIC_ONE_AGGREGATOR"
+    )]
+    aggregator: Option<bool>,
```

Comment:
I'm trying to think if this should be a state enum instead of a boolean, but I'm having a hard time thinking about what other modes the aggregator might have in the future.  I guess since I can't think of anything than just a bool is probably fine

Reply:
The plan is also to remove this boolean once we make it always on. We will likely need some form of config for which parts of the pipeline to enable. That will likely be a net new config option once we have move than one stage.

Reply:
should the name of the flag be `enable_aggregator` maybe?